manifests/v1beta1/components/controller/rbac.yaml

```diff
@@ -49,6 +49,15 @@ rules:
       - pods/status
     verbs:
       - "get"
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - "get"
+      - "list"
+      - "watch"
+      - "update"
```

Comment:
Any specific reason to make update instead of patch for the secret ?

Reply:
Ideally, we should use SSA. Let's replace UPDATE and PATCH with SSA in the cert-generator!

Reply:
Introducing SSA: https://github.com/kubeflow/katib/pull/2185/commits/748cc31757aacc44ddc11f7a3c2412e7811042af